### DISCUSSION TITLE
[Bug]: Notifications says success, but no files uploaded

### DISCUSSION THREAD

**Discussion by ArneNostitz** [ObsidianPublisher/obsidian-github-publisher#82] [Q&A — Plugin] [answered]
### Issue validation

- [X] My issue is not a question relative to [Material Mkdocs](https://squidfunk.github.io/mkdocs-material/) nor [Mkdocs](https://www.mkdocs.org/) (*[You can open a discussion to get more help!](https://github.com/squidfunk/mkdocs-material/discussions)*)
- [X] I checked the issue to prevent duplicate
- [X] I checked my configurations files and the documentation

### Workflow

Obsidian plugin 💠

### Describe the bug

I configured the plugin as described; test connection is successful. 

i added the "share: true" frontmatter to two files and start sync with "upload all files" from cmd+p menu (as rightclick menu does not get option even when activated)

the notification pops up saying its a success uploaded two files. but the developer info is full of errors. 

![Screenshot 2023-01-17 at 19 35 37](https://user-images.githubusercontent.com/16592596/212983171-a7733d69-935c-4647-8238-28c7922f5840.png)

### Configuration

{
  "githubRepo": "xcxcxc",
  "githubName": "xcxcxc",
  "GhToken": "xcxcxcxc",
  "githubBranch": "main",
  "shareKey": "share",
  "excludedFolder": [],
  "fileMenu": true,
  "editorMenu": true,
  "downloadedFolder": "fixed",
  "folderDefaultName": "test",
  "yamlFolderKey": "Project",
  "rootFolder": "",
  "workflowName": "",
  "embedImage": true,
  "defaultImageFolder": "",
  "autoCleanUp": false,
  "autoCleanUpExcluded": [],
  "folderNote": false,
  "folderNoteRename": "index.md",
  "convertWikiLinks": false,
  "convertForGithub": false,
  "subFolder": "",
  "embedNotes": false,
  "copyLink": true,
  "mainLink": "",
  "linkRemover": "",
  "hardBreak": false,
  "logNotice": false,
  "convertDataview": false,
  "useFrontmatterTitle": false,
  "censorText": [],
  "inlineTags": false,
  "dataviewFields": [],
  "excludeDataviewValue": [],
  "metadataFileFields": [],
  "frontmatterTitleKey": "title",
  "shareExternalModified": false,
  "automaticallyMergePR": true,
  "metadataExtractorPath": "",
  "convertInternalNonShared": false,
  "frontmatterTitleRegex": "",
  "frontmatterTitleReplacement": "",
  "ExcludedFolder": []
}

**Comment by Mara-Li** [comment, marked as answer]
check around the error you get : 
![82CE07A3-693E-43B4-AB6C-90B9735D6D51](https://user-images.githubusercontent.com/30244939/212983901-296f18bc-27dc-440c-a8c0-b335dcaff8c6.jpeg)

**Reply by Mara-Li** [reply]
https://stackoverflow.com/questions/68888746/getting-an-error-message-in-github-sorry-a-file-exists-where-you-re-trying-to

**Comment by ArneNostitz** [comment]
my bad... i had a file "test" on the repo and set the default folder to "test" as well ... bad habbit of putting "test" while "testing"...

**Reply by Mara-Li** [reply]
ahahah no worry ^^
